css/style.css

```diff
@@ -92,6 +93,161 @@ main {
   padding-right: 0.8rem;
 }
 
+.card-container {
+  display: grid;
+  grid-template-columns: 1.5rem 1fr 1.5rem;
+  grid-template-rows: 100%;
+  margin-top: 1rem;
+  margin-top: 3.625rem;
+}
+
+.card-wrapper {
+  grid-area: 1/2/1/2;
+  display: flex;
+  flex-direction: column;
+  border-radius: 1rem;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+}
+
+.card-image {
+  width: 100%;
+}
+
+.card-image > li > img {
+  object-fit: cover;
+  width: 100%;
+}
+
+.card-text {
+  margin-top: 0.75rem;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 2rem;
+  line-height: 2.75rem;
+}
+
+.card-text-container {
+  margin-top: 1rem;
+  font-size: 0.8125rem;
+  line-height: 1rem;
+  font-weight: 600;
+}
+
+.card-tagline {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.card-canopy {
+  color: #344563;
+}
+
+.card-job {
+  color: #7a869a;
+}
+
+.card-dot {
+  margin: 0.75rem;
+}
+
+.card-desc {
+  font-weight: 400;
+  font-size: 0.9rem;
+  line-height: 1.5rem;
+  margin-top: 1.25rem;
+  color: #344563;
+}
+
```

Comment:
- [ ] [OPTIONAL] Please make sure that the `font-size` of your `card-canopy` and `card-job` classes looks like that of the Figma design. This will make your text loom much easier to read and it is also the part of [project requirment](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

>  You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- [ ] Also please ☺️ decrease the margin between the `card-desc` and  `card-canopy`, `card-job` just like in figma design template.

**_Figma_**

<img width="201" alt="Screenshot 2023-02-25 at 8 08 02 AM" src="https://user-images.githubusercontent.com/44798044/221333253-a43d2ffc-53ff-4ad9-82f5-f6290807a70c.png">


_**Yours**_

<img width="306" alt="Screenshot 2023-02-25 at 7 34 11 AM" src="https://user-images.githubusercontent.com/44798044/221333262-b0c533fd-af68-4006-beb1-c6207497f6d9.png">